main.go

```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"github.com/apptreesoftware/protoc-gen-twirp_dart/generator"
+	"github.com/happy-co/protoc-gen-twirp_dart/generator"
```

Comment:
Don't change this - you should just use an alternate origin in your checkout from apptreesoftware

Reply:
Done, fixed the go deps as well.